updatecli/updatecli.d/jenkins-controllers/tools-jdk21.yaml

```diff
@@ -75,14 +75,14 @@ targets:
       file: config/jenkins_infra.ci.jenkins.io.yaml
       matchpattern: '_21(.*)\.(tar.gz|zip)"'
       replacepattern: '_{{ source `escapedVersion` }}.$2"'
-#     scmid: default
+    scmid: default
 
-# actions:
-#   default:
-#     kind: github/pullrequest
-#     scmid: default
-#     title: Bump JDK21 version (Jenkins tools) on all controllers to {{ source "lastVersion" }}
-#     spec:
-#       labels:
-#         - dependencies
-#         - jdk21
+actions:
+  default:
+    kind: github/pullrequest
+    scmid: default
+    title: Bump JDK21 version (Jenkins tools) on all controllers to {{ source "lastVersion" }}
+    spec:
+      labels:
+        - dependencies
+        - jdk21
```

Comment:
Same as the JDK17 above:

- Your change is a good catch!
- But we cannot merge it unless the manifest is fixed to generate the proper URLs